glambie/util/timeseries_helpers.py

```diff
@@ -548,6 +550,10 @@ def get_total_trend(start_dates, end_dates, changes, calculate_as_errors=False,
         input end dates of each time period
     changes : np.array or list
         input changes between start and end date
+    linear_regression : bool
+        If set to False, trends will be calculated as end_date minus start_date (i.e. mean of non-cumulative changes)
+        If set to True, trends will be calculated using a linear regression
+        By default False
```

Comment:
Seems this should accept an object of ExtractTrendsMethod type instead of a bool

Reply:
good point!